github/repos_projects.go

```diff
@@ -10,10 +10,19 @@ import (
 	"fmt"
 )
 
+// ProjectListOptions specifies the optional parameters to the
+// OrganizationsService.ListProjects and RepositoriesService.ListProjects methods.
+type ProjectListOptions struct {
+	// Indicates the state of the projects to return. Can be either open, closed, or all. Default: open
+	State string `url:"state,omitempty"`
```

Comment:
Ah, great.  Thank you, @shurcooL!
Then please totally disregard this comment, @shawncatz.